bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/discovery/HDPowerViewHubDiscoveryParticipant.java

```diff
@@ -43,7 +43,7 @@ public class HDPowerViewHubDiscoveryParticipant implements MDNSDiscoveryParticip
 
     private final Logger logger = LoggerFactory.getLogger(HDPowerViewHubDiscoveryParticipant.class);
 
-    private static final Pattern VALID_IP_V4_ADDRESS = Pattern
+    public static final Pattern VALID_IP_V4_ADDRESS = Pattern
```

Comment:
Consider extracting this to some common class so that `HDPowerViewDeviceDiscoveryService3` won't have to reference this class. Perhaps even `HDPowerViewBindingConstants`?

Reply:
Done.